### PR TITLE
config: correct PINOCCHIO_VERSION_UNKNOWN_TAG

### PR DESCRIPTION
**Pull request by jcarpent**

### CODE REVIEW

**Comment by jcarpent** [comment]
@gabrielebndn This solution is working well, without changing the existing code!

File: header.cmake [hunk 1]
```diff
@@ -147,20 +147,20 @@ ENDMACRO(_SETUP_PROJECT_HEADER)
 FUNCTION(GENERATE_CONFIGURATION_HEADER
     HEADER_DIR FILENAME LIBRARY_NAME EXPORT_SYMBOL)
 
-  IF(${PROJECT_VERSION_MAJOR} MATCHES UNKNOWN) 
-    SET(PROJECT_VERSION_MAJOR_CONFIG "${PROJECT_NAME}_VERSION_UNKNOWN_TAG")
+  IF(NOT ${PROJECT_VERSION_MAJOR} MATCHES UNKNOWN)
```
Comment (gabrielebndn) [inline header.cmake]: I'm sorry, why the `NOT`? I think it's a leftover, it should be removed, isn't it?

**Comment by gabrielebndn** [comment]
Better. This implies that version numbers may be signed, which may cause issues if somebody assumes `unsigned int`

**Comment by jcarpent** [comment]
Right, I would change then for 0, as you suggested, even if I'm favor in 0.0.0 version for first version.

**Pull request merged**